src/routers/productExports.ts

```diff
@@ -1,24 +1,171 @@
-
 import createRouter from '@/utilities/functions/createRouter';
+import * as ProdExportsService from '@/services/ProductExports';
+
 import authorized from '@/middleware/authorized';
 import { UserType } from '@/models/User/UserType';
 import { StatusCode } from '@/models/statusCodes';
 
 const router = createRouter();
 
+/** Represents a product export in the API */
+export interface ProdExport {
+  uuid: string;
+  transportId: string;
+  datetime: string;
+  productType: string;
+  amountOfProduct: number;
+  lastModified: string;
+}
+
+/**
+ * Represents a product export creation request in the API
+ */
+export interface ProdExportReq {
+  uuid: string;
+  transportId: string;
+  datetime: string;
+  productType: string;
+  amountOfProduct: number;
+}
+
+/**
+ * Represents a product exoprt udpate request in the API
+ */
+export interface ProdExportUpdateReq {
+  uuid: string;
+
+  transportId?: string;
+  datetime?: string;
```

Comment:
we probably shouldn't let them update the datetime for when it was first created?

Reply:
I wanted to give the clients more freedom. There's alot of things that I think shouldn't be changeable but seems like people wanted anything to be changed, including money transactions.